docs/user-guide/installation.rst

```diff
@@ -3,22 +3,40 @@ Installation
 
 ``tamr-unify-client`` is compatible with Python 3.6 or newer.
 
-----
+Stable releases
+---------------
 
 Installation is as simple as::
 
   pip install tamr-unify-client
 
+Or::
+
+  poetry add tamr-unify-client
+
 .. note::
-  We recommend you use a virtual environment for your project and install the
-  Python Client into that virtual environment.
+  If you don't use `poetry <https://poetry.eustace.io/>`_, we recommend you use a virtual environment for
+  your project and install the Python Client into that virtual environment.
 
   You can create a virtual environment with Python 3 via::
 
     python3 -m venv my-venv
 
   For more, see `The Hitchhiker's Guide to Python <https://docs.python-guide.org/dev/virtualenvs/>`_ .
 
+Latest (unstable)
```

Comment:
@mdonovan-tamr this is the relevant section for what you were trying to do when testing the geospatial features.